.idea/jarRepositories.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can you please remove this file from your git commit? You can delete it and commit again :)
As this is related to your local environment.